content/en/docs/concepts/overview/components.md

```diff
@@ -23,14 +23,48 @@ Here's the diagram of a Kubernetes cluster with all the components tied together
 {{% /capture %}}
 
 {{% capture body %}}
-## Control Plane Components
+## Node components
 
-The control plane's components make global decisions about the cluster (for example, scheduling), as well as detecting and responding to cluster events (for example, starting up a new {{< glossary_tooltip text="pod" term_id="pod">}} when a deployment's `replicas` field is unsatisfied).
+In Kubernetes, your application runs inside groups of one or more
+co-located containers called
+{{< glossary_tooltip text="_pods_" term_id="pod" >}}. Containers within a
+pod may share resources, such as an IP address and storage volumes, whereas
+pods are isolated from one another and from their hosts.  
+The hosts that run pods are called nodes, and one node can run pods from many
+different applications.
+
+To make this happen, Kubernetes runs several components on each node:
+
+### kubelet
+
+{{< glossary_definition term_id="kubelet" length="all" >}}
+
+### kube-proxy
+
+{{< glossary_definition term_id="kube-proxy" length="all" >}}
+
+### Container runtime
+
+{{< glossary_definition term_id="container-runtime" length="all" >}}
+
+## Control plane components
+
+The more different kinds of pod you want to run in your cluster, the
+more it helps to have systems to manage and share system resources,
```

Comment:
Double check these sentences. The first sentence is very generic. Could remove?
I think you could mention the API server?
workload-level?